### ISSUE TITLE
A11y_SQLAzureDataStudio_ServerDashboard_ServerDashboard_Keyboard: Image present under ‘Database Size’ section isn’t accessible using keyboard and alt attribute is missing for it

### ISSUE THREAD

**Issue by shweta255** [microsoft/azuredatastudio#9263]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#DesktopWeb;#Win10;#MAS1.1.1;#MAS2.1.1;#Benchmark;#A11y_SQLAzureDataStudio;#A11ySev2;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
Additional Details:
MAS reference: MAS1.1.1 and MAS2.1.1

**Repro Steps:**
1. Launch Azure Data Studio.
2. Connect to server.
3. In the Connections pane, right-click on a server & select Manage.
4. Launch NVDA and navigate to Toggle More button present under ‘Database Size’ section using tab key.
5. Try to navigate to the graph present below using arrow keys and Insert+ (4/6/8/2) keys

**Actual:**
Issue 1: Alt attribute is not present for the graphical image. 
Issue 2: When hovered mouse over any strip of pink/blue colour, tooltip is displayed on screen, but the same is not accessible using keyboard. 
**Same issue is observed for JAWS screenreader.**
**Same issue is observed in Database Dashboard screen:** 1. Launch Azure Data Studio.2. Connect to server.3. In the Connections pane, navigate to server --> database --> right click on any database & select Manage.

**Expected:**
**Issue 1:** Alt attribute should be present for the graphical image, so that when navigated to it, screenreader can announce description of the image.
**Issue 2:** Tooltips of any strip of pink/blue colour should be accessible using keyboard. 

**User Impact:**
Keyboard and Screen reader users will face difficulty to understand exactly what content is present under ‘Database Size’ section, as graph is neither accessible by keyboard nor it provides any alt attribute for screen reader users.

**Attachment link reference:**
[9263_A11y_SQLAzureDataStudio_ServerDashboard_ServerDashboard_Keyboard.zip](https://github.com/microsoft/azuredatastudio/files/4274873/9263_A11y_SQLAzureDataStudio_ServerDashboard_ServerDashboard_Keyboard.zip)

**Comment by alanrenmsft** [comment]
dupe of #9267

**Comment by sspatil123** [comment]
GithubTags:#Closed;Version 1.41.1
![image](https://user-images.githubusercontent.com/85863999/217254478-fd3f1dc0-9d95-4f1f-9165-d2c1d43347bc.png)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
